core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerBuilder.java

```diff
@@ -83,7 +84,10 @@ public CircuitBreakerBuilder(String name) {
 
     /**
      * Creates a new {@link CircuitBreakerBuilder}.
+     *
+     * @deprecated Use {@link CircuitBreaker#builder(String)}  instead.
```

Comment:
- `builder(String)` -> `builder()` ?
- `instead` can be omitted.

Reply:
Ok, I'll fix them all